extensions/amp-date-countdown/amp-date-countdown.md

```diff
@@ -0,0 +1,134 @@
+hours | i18n string for hour or hours
+minutes | i18n string for minute or minutes
+seconds | i18n string for second or seconds
+
+### Examples
+
+This table provides examples of formatted values specified in a Mustache template, and a sample of what the output:
+
+Format | Sample Output | Remarks
+-- | -- | -
+{hh}:{mm}:{ss} | 04:24:06 | -
+{h} {hours} and {m} {minutes} and {s} {seconds} | 4 hours and 1 minutes and 45 seconds | -
+{d} {days} {h}:{mm} | 1 day 5:03 | -
+{d} {days} {h} {hours} {m} {minutes} | 50 days 5 hours 10 minutes | -
+{d} {days} {h} {hours} {m} {minutes} | 20 days 5 hours 10 minutes | -
+{h} {hours} {m} {minutes} | 240 hours 10 minutes | `biggest-unit='hours'`
+{d} {days} {h} {hours} {m} {minutes} | 50 天 5 小时 10 分钟 | `locale='zh-CN'`
+
+
+## Attributes
+
+You must specify at least one of these required attributes: `end-date`, `timestamp-ms`, `timestamp-seconds`.
+
+##### end-date
+An ISO formatted date to count down to. For example, `2020-06-01T00:00:00+08:00`
+
+##### timestamp-ms
+A POSIX epoch value in milliseconds; assumed to be UTC timezone. For example, `timestamp-ms="1521880470000"`.
+
+##### timestamp-seconds
+A POSIX epoch value in seconds; assumed to be UTC timezone. For example, `timestamp-seconds="1521880470"`.
+
+##### offset-seconds (optional)
+A positive or negative number that represents the number of seconds to add or subtract from the `end-date`. For example, `offset-seconds="60"` adds 60 seconds to the end-date.
+
+##### when-ended (optional)
+Specifies whether to stop or continue the timer when it reaches 0 seconds. The value can be set to `stop` (default) or `continue`. If `stop`, the timer stops at 0 seconds and will not pass the final date.
+
+##### locale (optional)
+An i18n language string for each timer unit. The default value is `en` (for English).
+Supported values:
+
+Code | Language
+-- | --
+zh-CN | Chinese Simplified
+zh-TW | Chinese Traditional
+de | German
+en | English
+es | Spanish
+fr | French
+id | Indonesian
+it | Italian
+ja | Japanese
+ko | Korean
+nl | Dutch
+pt | Portuguese
+ru | Russian
+th | Thai
+tr | Turkish
+vi | Vietnamese
+
+##### biggest-unit (optional)
+Allows the `amp-date-countdown` component to calculate the time difference based on the specified `biggest-unit` value. For example, assume there are `50 days 10 hours` left, if the `biggest-unit` is set to `hours`, the result displays `1210 hours` left.
+
+* Supported values: `days`, `hours`, `minutes`, `seconds`
+* Default: `days`
+
+## Actions
+The `amp-date-countdown` exposes the following actions you can use [AMP on-syntax to trigger](https://www.ampproject.org/docs/reference/amp-actions-and-events):
+
+Action | Description
+-- | --
+`timeout` |  When the timer times out. For this action to function, `when-ended` **must be** set to `stop`. You can  define any actions when the timer times out. ![ezgif com-video-to-gif 2](https://user-images.githubusercontent.com/4065175/36954871-c05f8b0e-205f-11e8-944a-cbfff96fcb29.gif)
```

Comment:
I suggest you take the gif out of the table and create an "Example of action" section below the table, provide the gif and the code snippet.